src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -53,7 +53,9 @@ public AnalyzerExecutionContext(DiagnosticAnalyzer analyzer)
             /// </summary>
             private ImmutableArray<DiagnosticDescriptor> _lazyDescriptors = default(ImmutableArray<DiagnosticDescriptor>);
 
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
             public Task<HostSessionStartAnalysisScope> GetSessionAnalysisScopeTask(AnalyzerExecutor analyzerExecutor)
```

Comment:
This makes me think... should the analyzer bail out if the name of the method ends with Task? That might be an indiciation that the method name simply says it returns a Task, as opposed to being an async method.

Reply:
Hm. I don't want to sound too critical but that seems odd because in my experience from looking at PRs the compiler team is usually much quicker.

Reply:
😆 You have a fair point. However, since this PR relates to code style it follows a different approach to others.